entrypoint.sh

```diff
@@ -3,11 +3,19 @@ set -x
 
 REPOSITORY_NAME=$1
 REPOSITORY_BRANCH=$2
+run_npm_start=$3
+wait_for_url=$4
 
 git clone --branch $REPOSITORY_BRANCH https://github.com/$REPOSITORY_NAME.git /project-tests
 rm -rf /project-tests/.git
 cp -r /project-tests/* .
 npm install
+
+if $run_npm_start ; then
+  npm install -g wait-on
+  npm start & wait-on $wait_for_url
```

Comment:
em vez de usar o `npm install -g wait-on`, você poderia utilizar o `npx`:

```suggestion
  npm start & npx wait-on $wait_for_url
```